devspace.yaml

```diff
@@ -364,7 +385,78 @@ deployments:
       chart:
         name: kafka
         repo: https://charts.bitnami.com/bitnami
-        version: "21.0.0"
+        version: "26.0.0"
+      values:
+        # The mapping rules extract the CN part of the certificate to be used as the username in ACL rules
+        # kafka as a super user is the account used for provisioning and for inter-kafka communications.
+        # Other accounts have minimal rights defined in "extraProvisioningCommands"
+        extraConfig: |
+          authorizer.class.name = org.apache.kafka.metadata.authorizer.StandardAuthorizer
+          ssl.principal.mapping.rules = RULE:^.*OU=Stalker Kafka Clients.*,CN=(.*?)(,.*)?$/$1/,RULE:^CN=(.*?),OU=Stalker Kafka.*$/$1/,DEFAULT
+          super.users = User:kafka
+        listeners:
+          client:
+            name: CLIENT
+            containerPort: 9092
+            protocol: SSL
+            sslClientAuth: "required"
+          controller:
+            name: CONTROLLER
+            containerPort: 9093
+            protocol: SSL
+            sslClientAuth: "required"
+          interbroker:
+            name: INTERNAL
+            containerPort: 9094
+            protocol: SSL
+            sslClientAuth: "required"
+        tls:
+          passwordsSecret: kafka-jks-password
+          existingSecret: kafka-jks
+          keystorePassword: ${KAFKA_KEYSTORE_PASSWORD}
+          truststorePassword: ${KAFKA_KEYSTORE_PASSWORD}
+          endpointIdentificationAlgorithm: "https"
+        extraEnvVars:
+          - name: ALLOW_PLAINTEXT_LISTENER
+            value: "no"
+        provisioning:
+          enabled: true
+          auth:
+            tls:
+              type: jks
+              passwordsSecret: kafka-jks-password
+              certificatesSecret: kafka-jks
+              keystore: kafka.keystore.jks
+              truststore: kafka.truststore.jks
+          topics:
+            - name: stalker.jobs.findings
+              partitions: 1
+              replicationFactor: 1
+              config:
+                max.message.bytes: 64000
+                flush.messages: 1
+            - name: stalker.jobs.requests
+              partitions: 1
+              replicationFactor: 1
+              config:
+                max.message.bytes: 64000
+                flush.messages: 1
+            - name: stalker.jobs.logs
+              partitions: 1
+              replicationFactor: 1
+              config:
+                max.message.bytes: 64000
+                flush.messages: 1
```

Comment:
Should the topic names be configured as variables?

Reply:
Potentially, but it has code implication out of the scope of this PR for little gain for the end user. I don't see a use case of a user changing the kafka topics, it is not something that impacts them.